### PR TITLE
Constants Helper

### PR DESCRIPTION
**Pull request by StevenLangbroek**
This utility makes creating constants a little less verbose and error-prone. When writing tests for it, I noticed `electron-mocha` doesn't support (and probably never will support) `--watch`, see [#18](https://github.com/jprichardson/electron-mocha/pull/18), so I added `watch` as `devDependency` and a new task for running tests on file changes.
- Add watch as dependency for TDD, plus 'test:watch' task
- Add createConstants utility + specs

### CODE REVIEW

**Comment by rgbkrk** [comment]
:boom:

**Comment by rgbkrk** [comment]
That's clean!

**Comment by rgbkrk** [comment]
Looks like the app isn't launching after this, digging now.

File: src/actions/constants.js [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
+  'NEW_KERNEL', 'KILL_KERNEL',
+  'EXIT',
+  'SET_SELECTED',
+  'NEW_NOTEBOOK', 'READ_NOTEBOOK',
+  'MOVE_CELL', 'REMOVE_CELL',
+  'NEW_CELL_AFTER',
+  'UPDATE_CELL_EXECUTION_COUNT',
+  'UPDATE_CELL_OUTPUTS',
+  'UPDATE_CELL_SOURCE'
+);
```
Comment (rgbkrk) [inline src/actions/constants.js:16]: It occurs to me that we no longer get good autocomplete with the constants being generated directly.

If we switch it back to explicit exports, we get named exports to come up complete with type information:

screenshot 2016-03-03 22 38 05

Comment (StevenLangbroek) [inline src/actions/constants.js:16]: Well it doesn't know the type anymore. Is that important when working with this piece of code?

Comment (rgbkrk) [inline src/actions/constants.js:16]: All about hints and developer experience here. I'm not going to argue either way, I noticed the limitation and pointed it out.

**Pull request merged**